libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/AuthenticatorService.java

```diff
@@ -173,42 +174,40 @@ public Bundle getAuthToken(
                             Account account,
                             String authTokenType,
                             Bundle options) throws NetworkErrorException {
-            final String TAG = "Auth..Ser..:getAuthT..";
             final AccountManager mgr = AccountManager.get(context);
-            final String passcodeHash = SalesforceSDKManager.getInstance().getPasscodeHash();
-            final String refreshToken = SalesforceSDKManager.decryptWithPasscode(mgr.getPassword(account), passcodeHash);
-            final String loginServer = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_LOGIN_URL), passcodeHash);
-            final String clientId = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_CLIENT_ID), passcodeHash);
-            final String instServer = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_INSTANCE_URL), passcodeHash);
-            final String userId = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_USER_ID), passcodeHash);
-            final String orgId = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_ORG_ID), passcodeHash);
-            final String username = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_USERNAME), passcodeHash);
-            final String lastName = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_LAST_NAME), passcodeHash);
-            final String email = SalesforceSDKManager.decryptWithPasscode(mgr.getUserData(account, AuthenticatorService.KEY_EMAIL), passcodeHash);
+            final String refreshToken = SalesforceSDKManager.decrypt(mgr.getPassword(account));
```

Comment:
Replaced all `decryptWithPasscode...` methods in this file with `decrypt...` and `encryptWithPasscode...` methods with `encrypt...`.